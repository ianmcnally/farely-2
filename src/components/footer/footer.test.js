import React from 'react'
import Footer from './footer'
import renderShallow from 'render-shallow'

it('renders the correct footer information', () => {
  const component = renderShallow(<Footer />).output
  expect(component).toMatchSnapshot()
})
