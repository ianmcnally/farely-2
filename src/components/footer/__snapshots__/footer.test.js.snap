exports[`test renders the correct footer information 1`] = `
<footer
  className="f6 bt b--black-20">
  <p>
    Say hello 
    <a
      href="twitter.com/farelyapp">
      @farelyapp
    </a>
  </p>
</footer>
`;
